spec/models/offer_spec.rb

```diff
@@ -5,6 +5,7 @@
     it { is_expected.to have_db_column :helper_id }
   end
 
+    it { is_expected.to have_db_column :status }
```

Comment:
This should probably go in the database table block

Reply:
Moved!